deploy/azure/key-vault/key-rotation/service-bus.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# set errors on
+set -e
+
+# renews key for service bus and updates key vault
+
+# input parameters
+RES_GROUP=$1 # target resource group
+KEY_VAULT_NAME=$2 # key vault instance
+SERVICE_BUS_NAME=$3 # service bus instance
+KEY_TYPE=$4 # accepted values are 'primary' or 'secondary'
+SERVICE_BUS_USER=$5 # the target user for key rotation
+
+SERVICE_BUS_KEY_TYPE="" # valid values are 'PrimaryKey', 'SecondaryKey'
+KEY_IDENTIFIER="" # valid valuees are 'primaryConnectionString', 'secondaryConnectionString'
+
+# Azure CLI uses slightly different identifiers for key type across different services :-/
+# align them for each service...
+if [ "$KEY_TYPE" = "primary" ]
```

Comment:
There is a deviation from the previous standard where the `if` statement was terminated by a semi-colon.  I think it is better if we maintain consistency. 

Having the semi-colon seems like the more robust code to me. As it allows the `then` keyword to be on the same line as the `if`, which is a common format.